examples/notification-handler.php

```diff
@@ -1,12 +1,31 @@
 <?php
+// This is just for very basic implementation reference, in production, you should validate the incoming requests and implement your backend more securely.
+// Please refer to this docs for sample HTTP notifications:
+// https://docs.midtrans.com/en/after-payment/http-notification?id=sample-of-different-payment-channels
 
 namespace Midtrans;
 
 require_once dirname(__FILE__) . '/../Midtrans.php';
 Config::$isProduction = false;
-Config::$serverKey = '<your serverkey>';
-$notif = new Notification();
+Config::$serverKey = '<your server key>';
+if (strpos(Config::$serverKey, 'your ') != false ) {
+    echo "<code>";
+    echo "<h4>Please set your server key from sandbox</h4>";
+    echo "In file: " . __FILE__;
+    echo "<br>";
+    echo "<br>";
+    echo htmlspecialchars('Config::$serverKey = \'<your server key>\';');
+    die();
+}
```

Comment:
This block is not part of the example. Can be confusing to read this part of code. Can you make it a function that is defined in a bottom?

e.g. at the bottom of  this file:
```
// non-relevant function only used for demo/example purpose
function printExampleWarningMessage(){
...
}
```
Then call it on line 11.

Also within that function check if method is not POST, then print warning: 
Notification-handler are not meant to be opened via browser / GET HTTP method. It is used to handle Midtrans HTTP POST notification / webhook. <put link to docs about notif handling>